### PR TITLE
Use kubekins-e2e in kwok CI

### PR DESCRIPTION
**Pull request by wzshiming**
Use the same image as the other CIs

https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Dockerfile

/cc @Huang-Wei

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml [hunk 1]
```diff
@@ -12,10 +12,11 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - image: golang:1.18
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220804-4fa19ea91a-master
```
Comment (Huang-Wei) [inline config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml:15]: can we keep non-e2e jobs still using a plain golang image?

File: config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml [hunk 2]
```diff
@@ -76,8 +83,12 @@ presubmits:
...
```
Comment (Huang-Wei) [inline config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml:86]: this looks good to me, but guess using kubekins-e2e for other jobs is an overkill?

**Comment by wzshiming** [comment]
@Huang-Wei 
Updated

**Comment by wzshiming** [comment]
@Huang-Wei 
I just missed something and clean it, let's do it again LGTM please.

**Pull request merged**